plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -903,6 +904,30 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
         }
     }
 
+    @Override
+    public void testNativeQuerySelectFromTestTable()
+    {
+        // override because Postgres needs explicit schema
```

Comment:
You can use `getSession().getSchema().orElseThrow()` to avoid override.
(Trino session schema = remote schema name (usually))

also, nit: Postgres -> PostgreSQL